java/plasma/src/main/java/org/apache/arrow/plasma/PlasmaClient.java

```diff
@@ -46,19 +46,19 @@ public PlasmaClient(String storeSocketName, String managerSocketName, int releas
 
   // interface methods --------------------
 
+  /**
+   * don't catch exception, exception dealt with in raylet
```

Comment:
`don't catch exception, exception dealt with in raylet`
This should appear here, because ray is only a user of plasma client and plasma client shouldn't be aware of it.
Just describe what this function does in the function document.